tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py27-django{18,110,111}
+
+[testenv]
+deps =
+    -r{toxinidir}/requirements/requirements.txt
+    -r{toxinidir}/requirements/requirements-test.txt
+    django18: Django>=1.8,<1.9
+    django110: Django>=1.10,<1.11
```

Comment:
Although EOL'd, I recommend adding Django 1.9 support in case that is needed as an upgrade path.